exchanges/cryptodotcom/cryptodotcom_wrapper.go

```diff
@@ -605,17 +672,32 @@ func (cr *Cryptodotcom) SubmitOrder(ctx context.Context, s *order.Submit) (*orde
 		// For MARKET (BUY), STOP_LOSS (BUY), TAKE_PROFIT (BUY) orders only: Amount to spend
 		notional = s.Amount
 	}
-	ordersResp, err := cr.CreateOrder(ctx, &CreateOrderParam{
-		InstrumentName: format.Format(s.Pair),
-		Side:           s.Side,
-		OrderType:      s.Type,
-		Price:          s.Price,
-		Quantity:       s.Amount,
-		ClientOrderID:  s.ClientOrderID,
-		Notional:       notional,
-		PostOnly:       s.PostOnly,
-		TriggerPrice:   s.TriggerPrice,
-	})
+	var ordersResp *CreateOrderResponse
+	if cr.Websocket.CanUseAuthenticatedWebsocketForWrapper() {
+		ordersResp, err = cr.WsPlaceOrder(&CreateOrderParam{
+			InstrumentName: format.Format(s.Pair),
+			Side:           s.Side,
+			OrderType:      orderTypeToString(s.Type),
+			Price:          s.Price,
+			Quantity:       s.Amount,
+			ClientOrderID:  s.ClientOrderID,
+			Notional:       notional,
+			PostOnly:       s.PostOnly,
+			TriggerPrice:   s.TriggerPrice,
+		})
```

Comment:
create a variable for CreateOrderParam pointer above so it doesn't need to be declared twice.